documentation/src/main/asciidoc/chapter-3-defining-a-mapper.asciidoc

```diff
@@ -537,8 +537,10 @@ When doing a mapping MapStruct checks if there is a builder for the type being m
 If there is no builder, then MapStruct looks for a single accessible constructor.
 When there are multiple constructors then the following is done to pick the one which should be used:
 
-* If a parameterless constructor exists then it would be used to construct the object, and the other constructors will be ignored
-* If there are multiple constructors then the one annotated with annotation named `@Default` (from any package) will be used
+* If a constructor is annotated with an annotation named `@Default` (from any package) it will be used.
+* If a single public constructor exists then it will be used to construct the object, and the other non public constructors will be ignored.
+* If a parameterless constructor exists then it will be used to construct the object, and the other constructors will be ignored.
+* If there are multiple eligible constructors then there will be a compilation error due to ambigious constructors. In order to break the ambiquity an annotation named `@Default` (from any package) can used.
```

Comment:
Perhaps add an example?

Reply:
That's a good point. I'll add one tonight